parseorders.cpp

```diff
@@ -1626,7 +1626,6 @@ void Game::ProcessBuildOrder(Unit *unit, AString *o, OrdersCheck *pCheck)
 						obj->SetName(new AString("Building"));
 						unit->build = obj->num;
 						unit->object->region->objects.Add(obj);
-						unit->MoveUnit(obj);
```

Comment:
we can remove it because it happens inside BUILD orders https://github.com/artyomtrityak/Atlantis/blob/master/monthorders.cpp#L661